src/js/common/schemaform/widgets/TextWidget.jsx

```diff
@@ -13,7 +13,7 @@ export default function TextWidget(props) {
         maxLength={props.schema.maxLength}
         autoComplete={props.options.autocomplete || false}
         className={props.options.widgetClassNames}
-        value={props.value}
+        value={props.value || ''}
```

Comment:
The original change from `value={props.value || ''}` to `value={props.value}` was for testing, correct? Was there any other reason to allow `undefined` instead of a blank string? Will it cause blank strings to be sent back to the back end when `undefined` (or nothing at all) should be?